docs/addingTests.md

```diff
@@ -0,0 +1,33 @@
+## Adding Tests
+1) Create a new test class that extends `AbstractIntegrationTest` in `src/integTest/groovy/net/wasdev/wlp/gradle/plugins`
+2) Add a directory in `src/integTest/resources` that contains the files needed to build the project you wish to test against
+3) In your test class, add the `sourceDir` property to create a reference to your test resources. Example:
+
+```groovy
+static File sourceDir = new File("build/resources/integrationTest/liberty-test")
+```
+
+4) In your test class, add the following boilerplate code to your `@BeforeClass` method to ensure your test resources are loaded during test execution:
+
+```groovy
+    @BeforeClass
+    public static void setup() {
+        deleteDir(integTestDir)
+        createDir(integTestDir)
+        if(test_mode == "offline"){
+            WLP_DIR.replace("\\","/")
```

Comment:
WLP_DIR in the location of a local liberty install. I'm certain why this transformation is here... I alway run online.